src/main/java/org/alfresco/repo/search/impl/lucene/SolrActionStatusResult.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2016 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software. 
+ * If the software was purchased under a paid Alfresco license, the terms of 
+ * the paid license agreement will prevail.  Otherwise, the software is 
+ * provided under the following open source license terms:
+ * 
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.repo.search.impl.lucene;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * The results of executing a SOLR STATUS action
+ *
+ * @author aborroy
+ * @since 6.2
+ */
+public class SolrActionStatusResult implements JSONAPIResult
+{
+    private static final Log logger = LogFactory.getLog(SolrActionStatusResult.class);

```

Comment:
Generally I'd use SLF4J for new loggers, and put static final in all caps, like `LOGGER`.  There are some advantages of using SLF4J - mainly the lazy string construction:
```
LOGGER.debug("JSON response: {}", response);
```

Reply:
Moving all loggers to SLF4J.